InvenTree/common/api.py

```diff
@@ -133,6 +138,29 @@ def get(self, request, format=None):
         return Response(response)
 
 
+class CurrencyRefreshView(APIView):
+    """API endpoint for manually refreshing currency exchange rates.
+
+    User must be a 'staff' user to access this endpoint
+    """
+
+    permission_classes = [
+        permissions.IsAuthenticated,
+        permissions.IsAdminUser,
+    ]
+
+    def post(self, request, *args, **kwargs):
+        """Performing a POST request will update currency exchange rates"""
+
+        from InvenTree.tasks import update_exchange_rates
+
+        update_exchange_rates
```

Comment:
Question @SchrodingersGat : should this call the function or just reference it?

Reply:
Uh, definitely call. I'll fix that.